<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditFifthsDialog</class>
 <widget class="QDialog" name="EditFifthsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit scale by fifths</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboScale">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Diatonic only (7 keys)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chromatic (12 keys)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Simple Enharmony (17 keys)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Simple Enharmony (21 keys)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Full Enharmony (35 keys)</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="editFifthsLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit scale as fifths:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of notes for scale:</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>181</width>
     <height>291</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>179</width>
      <height>289</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type of values shown:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboStoringMode">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>absolute cent values</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>delta cent from equal temperament (12 EDO)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>interval as frequency ratio</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditFifthsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditFifthsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
